packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/04-upgradeable-plugin/05-updating-versions.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Upgrading an Upgradeable Plugin
+---
+
+## How to upgrade an Upgradeable Plugin
+
+Updating an Upgradeable plugin means we want to direct the implementation of our functionality to a new build, rather than the existing one.
+
+In this tutorial, we will go through how to update an Upgradeable plugin and each component needed.
+
+### 1. Create the new build implementation contract
+
+Firstly, you want to create the new build implementation contract the plugin should use. You can read more about how to do this in ["How to create a subsequent build implementation to an Upgradeable Plugin" guide](./04-subsequent-builds.md).
+
+```solidity
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.17;
+
+import {IDAO, PluginUUPSUpgradeable} from '@aragon/osx/core/plugin/PluginUUPSUpgradeable.sol';
+
+/// @title SimpleStorage build 2
+contract SimpleStorageBuild2 is PluginUUPSUpgradeable {
+  bytes32 public constant STORE_PERMISSION_ID = keccak256('STORE_PERMISSION');
+
+  uint256 public number; // added in build 1
+  address public account; // added in build 2
+
+  /// @notice Initializes the plugin when build 2 is installed.
+  function initializeBuild2(
```

Comment:
Same, only one global initializer per plugin version and one global updater:

https://github.com/aragon/simple-storage-example-plugin/blob/main/contracts/release1/build3/SimpleStorageR1B3.sol#L30-L50

Reply:
Simple storage plugin update - added to notes!